ms2pip/ms2pipC.py

```diff
@@ -946,10 +939,18 @@ def run(self):
 
         self._read_peptide_information()
 
+        if self.add_retention_time:
+            logging.info("Adding retention time predictions")
+            if self.config_parser:
+                rt_predictor = RetentionTime(config=self.config_parser.config)
+            else:
+                rt_predictor = RetentionTime()
+            rt_predictor.add_rt_predictions(self.data)
```

Comment:
Not something that should be done in this PR, but one option to improve the runner class and method would be to split the different logical actions to different methods, i.e. instead of having a `match_spectrum` option, or deferring actions from passed MGF files.